app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCStatus.scala

```diff
@@ -325,24 +223,6 @@ object DLCStatus {
     override val statusString: String = "REMOTE CLAIMED"
   }
```

Comment:
This class should have a method which computes the `oracleSig` from `cet` and adaptor signature from setup, also `closingTx` should be removed from both this and the local version